components/_typography/text.scss

```diff
@@ -175,20 +175,41 @@
   @extend .msds-text-transform-uppercase;
 }
 
+.msds-text-display-1-regular {
```

Comment:
Why not just add a modifier to the display-1 class and then overwrite / add the things you need in there, instead of adding them twice in 2 different classes?

Reply:
I followed the convention for the other font stylings, to maintain consistency